README.md

```diff
@@ -2,36 +2,63 @@
 
 ## Goals
 
-This script is for fingerprinting servers to determine what vunerabilities should lead to responsible disclosure. 
-
-
-## TO DO
-0. security.txt = done (shodan)
-1. Find a domain
-- ssl subject or issuer domain (exclude common CAs)
-- check other ports on same IP (shodan)
-    - EHLO banner
-    - web content
-    - ssh banner
-    - SNMP
-- passive dns domain (dumpsterDNS, circl.lu etc)
-- reverse dns domain (exclude answers that contain the ip address in reverse as prob just the ISP?)
-- check BGP and repeat for other IPs in the subnet, find a pattern?
+This script is for fingerprinting servers to find potential vunerabilities for responsible disclosure. 
```

Comment:
No, this is not to find vulns. This script is just meant to find the owners of vulnerable systems in bulk so we can contact them.